gcloud/monitoring/client.py

```diff
@@ -195,6 +203,121 @@ def metric_descriptor(self, type_,
             display_name=display_name,
         )
 
+    @staticmethod
```

Comment:
What's wrong with just using

```python
from gcloud import monitoring
metric = monitoring.Metric(type_, labels)
```

in your examples? We already do the same for `datastore.Entity`, which doesn't depend on any client.